Makefile

```diff
@@ -4,8 +4,12 @@ PHONY: clean
 clean:
 	-rm -r dist/
 
-PHONY: build
-build:
-	tools/make-firefox.sh
+build: dist/build/uMatrix.chromium dist/build/uMatrix.firefox
```

Comment:
Actually `build`  should be marked as PHONY target since it is not a file name.

Reply:
/me facepalm
will add back if reopen